modules/adnowBidAdapter.js

```diff
@@ -0,0 +1,83 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { NATIVE, BANNER } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+
+const BIDDER_CODE = 'adnow';
+const ENDPOINT = '//n.ads3-adnow.com/a';
+
+/** @type {BidderSpec} */
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [ NATIVE, BANNER ],
+
+  /**
+   * @param {object} bid
+   * @return {boolean}
+   */
+  isBidRequestValid: (bid) => {
+    return !!(bid && bid.params && (bid.params.codeId));
+  },
+
+  /**
+   * @param {BidRequest[]} bidRequests
+   * @param {*} bidderRequest
+   * @return {ServerRequest}
+   */
+  buildRequests: (bidRequests, bidderRequest) => {
+    return bidRequests.map(req => {
+      return {
+        method: 'GET',
+        url: ENDPOINT,
+        data: utils.parseQueryStringParameters({
+          Id: req.params.codeId,
+          mediaType: req.params.mediaType || 'native',
+          out: 'prebid',
+          d_user_agent: navigator.userAgent,
+          requestid: req.bidId
+        }),
+        options: {
+          withCredentials: false,
+          crossOrigin: true
+        },
+        bidRequest: req
+      };
+    });
+  },
+
+  /**
+   * @param {*} response
+   * @param {ServerRequest} request
+   * @return {Bid[]}
+   */
+  interpretResponse: (response, request) => {
+    const bidObj = request.bidRequest;
+    let bid = response.body;
```

Comment:
You'll need to follow the [prebid docs interpret response instructions](http://docs.prebid.org/dev-docs/bidder-adaptor.html#interpreting-the-response) and build a bid object from your server response with the required parameters listed and any optional you will need for your mediatype.  It cannot be passed through straight from your ajax response.